launchable/commands/subset.py

```diff
@@ -635,32 +670,32 @@ def run(self):
                 ],
                 [
                     "Remainder",
-                    len(original_rests),
+                    len(original_rest),
                     summary["rest"].get("rate", 0.0),
                     summary["rest"].get("duration", 0.0),
                 ],
                 [],
                 [
                     "Total",
-                    len(original_subset) + len(original_rests),
+                    len(original_subset) + len(original_rest),
                     summary["subset"].get("rate", 0.0) + summary["rest"].get("rate", 0.0),
                     summary["subset"].get("duration", 0.0) + summary["rest"].get("duration", 0.0),
                 ],
             ]
 
-            if is_brainless:
+            if subset_result.is_brainless:
                 click.echo(
                     "Your model is currently in training", err=True)
 
             click.echo(
                 "Launchable created subset {} for build {} (test session {}) in workspace {}/{}".format(
-                    subset_id,
+                    subset_result.subset_id,
                     build_name,
                     test_session_id,
                     org, workspace,
                 ), err=True,
             )
-            if is_observation:
+            if subset_result.is_observation:
```

Comment:
Why is `is_observation` a response from the server? How is the different from the local flag `this.is_observation`?

Reply:
Ah, I have the answer myself. The observation flag is an attribute of a test session, not that of a subset request. Yikes.

Reply:
Ah, I have the answer myself. The observation flag is an attribute of a test session, not that of a subset request. Yikes.

Reply:
Yes, right